crates/api/src/local_user/login.rs

```diff
@@ -89,15 +89,12 @@ async fn check_registration_application(
     && !local_user_view.local_user.accepted_application
     && !local_user_view.local_user.admin
   {
-    // Fetch the registration, see if its denied
+    // Fetch the registration application. If no admin id is present its still pending. Otherwise it
+    // was processed (either accepted or denied).
     let local_user_id = local_user_view.local_user.id;
     let registration = RegistrationApplication::find_by_local_user_id(pool, local_user_id).await?;
-    if let Some(deny_reason) = registration.deny_reason {
-      let lang = utils::get_interface_language(local_user_view);
-      let registration_denied_message = format!("{}: {}", lang.registration_denied(), deny_reason);
-      Err(LemmyErrorType::RegistrationDenied(
-        registration_denied_message,
-      ))?
```

Comment:
I think its better to leave it to frontends to format the message here. Anyway this wasnt working properly because the registration_denied string is missing a placeholder, so it didnt include deny_reason. Also lemmy-ui seems to ignore the deny_reason response completely.